lib/easypost/util.rb

```diff
@@ -125,7 +131,8 @@ def self.get_lowest_smart_rate(smart_rates, delivery_days, delivery_accuracy)
     lowest_smart_rate = nil
 
     unless valid_delivery_accuracy_values.include?(delivery_accuracy.downcase)
-      raise EasyPost::Error.new("Invalid delivery accuracy value, must be one of: #{valid_delivery_accuracy_values}")
+      suggestion = "Must be one of: #{valid_delivery_accuracy_values}"
+      raise EasyPost::Errors::InvalidParameterError.new('delivery_accuracy', suggestion)
```

Comment:
suggestion: inline the suggestion
```suggestion
      raise EasyPost::Errors::InvalidParameterError.new(
          'delivery_accuracy',
          "Must be one of: #{valid_delivery_accuracy_values}"
      )
```